src/org/rust/lang/RustSdkType.kt

```diff
@@ -27,6 +29,10 @@ class RustSdkType : SdkType("Rust SDK") {
             val homebrew = File("/usr/local/Cellar/rust")
             if (homebrew.exists())
                 return homebrew.absolutePath
+            val multiRust = File(FileUtil.expandUserHome("~/.multirust/toolchains"))
+            if (multiRust.exists()) {
+                return multiRust.absolutePath
+            }
```

Comment:
IMHO this should be in front of the homebrew code. Multirust seem more important to me than the homebrew installation.